test/corpus/preprocessor.txt

```diff
@@ -13,39 +13,32 @@ If, elif and else directives
 --------------------------------------------------------------------------------
 
 (compilation_unit
-  (if_directive
-    (identifier))
-  (global_statement
+  (preproc_if
```

Comment:
Let's see how we get on with these if > preproc changes - they're not part of Roslyn and probably less used by third party tooling relying on this library.